media/static-bundles.json

```diff
@@ -1452,6 +1461,14 @@
         "js/products/vpn/invite.js"
       ],
       "name": "mozilla-vpn-invite"
+    },
+    {
+      "files": [
+        "js/base/mozilla-fxa-product-button.js",
+        "js/products/vpn/geo.js",
```

Comment:
Are these needed? The pages just seem to link to the landing page so may not be necessary.